prow/plugins.yaml

```diff
@@ -72,101 +69,117 @@ approve:
   implicit_self_approve: true
 
 plugins:
-  istio/istio:
-  - approve
-  - assign
-  - blunderbuss
-  - golint
-  - help
-  - hold
-  - lgtm
-  - lifecycle
-  - slackevents
-  - trigger
-  - verify-owners
-  - wip
-
   istio/api:
-  - approve
-  - assign
-  - blunderbuss
-  - golint
-  - help
-  - hold
-  - lgtm
-  - lifecycle
-  - slackevents
-  - trigger
-  - verify-owners
-  - wip
+    - needs-rebase
+    - size
+    - trigger
+
+  istio/bots:
+    - needs-rebase
+    - size
+    - trigger
+
+  istio/cni:
+    - needs-rebase
+    - size
+    - trigger
+
+  istio/common-files:
+    - needs-rebase
+    - size
+    - trigger
 
   istio/installer:
-  - trigger
+    - needs-rebase
+    - size
+    - trigger
+
+  istio/istio:
+    - approve
+    - assign
+    - blunderbuss
+    - golint
+    - help
+    - hold
+    - lgtm
+    - lifecycle
+    - slackevents
+    - trigger
+    - verify-owners
+    - wip
+
+  istio/istio.io:
+    - needs-rebase
+    - size
+    - trigger
 
   istio/operator:
-  - assign
-  - hold
-  - slackevents
-  - trigger
-  - wip
+    - needs-rebase
+    - size
+    - trigger
 
-  istio/proxy:
-  - approve
-  - assign
-  - blunderbuss
-  - golint
-  - help
-  - hold
-  - lgtm
-  - lifecycle
-  - slackevents
-  - trigger
-  - verify-owners
-  - wip
+  istio/pkg:
+    - needs-rebase
```

Comment:
I think these need to be under external plugins

Reply:
I didn't change these, I just moved them around to alphabetize.

Reply:
Yep. Fixed